### PR TITLE
SE-36-add-additional-units-to-concrete-input-conversions

### PR DESCRIPTION
**Pull request by reeves999999**
Added more unit types

### CODE REVIEW

File: src/utils/units.js [hunk 1]
```diff
@@ -18,51 +17,70 @@ const conversions = {
...
+  "kN/m": "N/m",
+  N: "N",
+  kN: "N",
+  "kN/m³": "N/m³",
+  "kN/m²": "N/m²",
+  K: "K",
+  "W/m*K": "W/m*K",
+  MN: "MN",
+  kNm: "kNm",
+  MNm: "MNm",
```
Comment (YT8434) [inline src/utils/units.js:77]: Is SI unit for stress not N/m²?

File: src/utils/units.js [hunk 2]
```diff
@@ -75,8 +75,8 @@ const SI = {
   MN: 'N',
   kNm: 'Nm',
   MNm: 'Nm',
-  'N/mm2': 'N/mm2',
-  MPa: 'N/mm2',
+  'N/mm²': 'N/m',
+  MPa: 'N/m',
```
Comment (YT8434) [inline src/utils/units.js:79]: N/m² instead of N/m for stress unit

File: src/utils/units.js [hunk 3]
```diff
@@ -59,6 +72,11 @@ const SI = {
   'kN/m²': 'N/m²',
   K: 'K',
   'W/m*K': 'W/m*K',
+  MN: 'N',
+  kNm: 'Nm',
+  MNm: 'Nm',
+  'N/mm²': 'N/m²',
+  MPa: 'N/m',
```
Comment (danielstorey) [inline src/utils/units.js:79]:
```suggestion
  MPa: 'N/m²',
```

File: src/utils/units.js [hunk 4]
```diff
@@ -75,7 +75,7 @@ const SI = {
   MN: 'N',
   kNm: 'Nm',
   MNm: 'Nm',
-  'N/mm²': 'N/m',
+  'N/mm²': 'N/m²',
   MPa: 'N/m',
```
Comment (YT8434) [inline src/utils/units.js:79]: MPa: 'N/m²'

**Review by danielstorey — CHANGES_REQUESTED** [review]

File: src/utils/units.js (hunk 4 above)
Comment (reeves999999) [reply src/utils/units.js]: Please check latest.

File: src/utils/units.js (hunk 2 above)
Comment (reeves999999) [reply src/utils/units.js]: Please check latest.

File: src/utils/units.js (hunk 1 above)
Comment (reeves999999) [reply src/utils/units.js]: Please check latest.

**Review by YT8434 — APPROVED** [review]

**Review by danielstorey — APPROVED** [review]

**Pull request merged**